content/body/reflow.php

```diff
@@ -119,7 +119,7 @@
 <h2>Common Reflow Problem #2: Filters on Product Listing Pages</h2> 
 
 <p>
-  A common design problem on many e-commerce sites is on PLPs (product listing pages). Consider the screenshot below
+  A common design problem on many e-commerce sites is PLPs (product listing pages). Consider the screenshot below
```

Comment:
I think the original text made more sense:

```suggestion
  A common design problem on many e-commerce sites is on PLPs (product listing pages). Consider the screenshot below
```